### PR TITLE
Drop JAXB from CertReviewResponse

### PR DESCRIPTION
**Pull request by edewata**
This PR is created on top of PR #3711.

### CODE REVIEW

File: base/common/src/main/java/com/netscape/certsrv/cert/CertEnrollmentRequest.java [hunk 1]
```diff
@@ -349,18 +359,209 @@ public boolean equals(Object obj) {
...
+        for (ProfileInput input: getInputs()) {
+            certEnrollmentRequestElement.appendChild(input.toDOM(document));
+        }
+
+        for (ProfileOutput output: getOutputs()) {
+
+            certEnrollmentRequestElement.appendChild(output.toDOM(document));
+
+            NodeList nodes = certEnrollmentRequestElement.getElementsByTagName("profileOutput");
+            System.out.println(nodes.getLength());
```
Comment (ckelleyRH) [inline base/common/src/main/java/com/netscape/certsrv/cert/CertEnrollmentRequest.java:409]: I think it is good but for these debug statements, which should drop away when rebasing on #3711.

**Review by ckelleyRH — APPROVED** [review]
LGTM!

File: base/common/src/main/java/com/netscape/certsrv/cert/CertReviewResponse.java [hunk 2]
```diff
@@ -290,4 +258,246 @@ public boolean equals(Object obj) {
                 && Objects.equals(requestStatus, other.requestStatus) && Objects.equals(requestType, other.requestType);
     }
 
+    public Element toDOM(Document document) {
+
+        Element element = document.createElement("certReviewResponse");
```
Comment (c-dorney) [inline base/common/src/main/java/com/netscape/certsrv/cert/CertReviewResponse.java:263]: I did a diff between the results or the unit test on master and your branch and an upper case C was one of the differences 
```suggestion
        Element element = document.createElement("CertReviewResponse");
```

**Review by c-dorney — APPROVED** [review]
Looks good to me, I had one comment but it wasn't valid so I removed it!

Definitely worth rebasing with #3711 when that is merged.

**Comment by ckelleyRH** [comment]
Looks like it will have to be rebased anyway to resolve a conflict.

**Comment by edewata** [comment]
Thanks! I've rebased the PR.

**Pull request merged**